index.js

```diff
@@ -28,14 +28,16 @@ var userData = app.getPath("userData");
 var configPath = path.join(userData, "config.json");
 var serversPath = path.join(userData, "servers.json");
 var versionsPath = path.join(userData, "versions.json");
+var hashPath = path.join(userData, "hashes.json");
 
 function initialSetup(firstTime) {
     if (!firstTime) {
-        // Migration from pre-1.4
+        // Migration from pre-1.6
         // Back everything up, just in case
         fs.copySync(configPath, configPath + ".bak");
         fs.copySync(serversPath, serversPath + ".bak");
         fs.copySync(versionsPath, versionsPath + ".bak");
+        fs.copySync(hashPath, hashPath + ".bak");
```

Comment:
Would this file exist in versions prior to 1.6?

Reply:
No, the file should be new. We don't need this line yet actually, but we might forget to do it for 1.7+. Since its this PR's new file, I thought it was best if this PR handled it.

Reply:
Yep I agree that we should have it for further versions, will this throw an error if hashPath doesn't exist though?

Reply:
Oh you're correct to point that out. It's been a while and I'm sure I encountered this error as well, but somehow it slipped through the cracks. I added if checks here now.